proto/vote/exported/v1beta1/types.proto

```diff
@@ -25,6 +25,8 @@ message Voter {
   int64 voting_power = 2;
 }
 
+message Vote { repeated google.protobuf.Any results = 1; }
```

Comment:
Are we voting on multiple things at the same time?

Reply:
yes